### PR TITLE
Changes in validation constraints messages

### PR DESCRIPTION
**Pull request by Subbu-betalectic**

### CODE REVIEW

File: app/requests/base.js [hunk 1]
```diff
@@ -34,11 +34,15 @@ module.exports = (constraints, req, res, next) => {
           var response = {
             message: `Validation failed. ${validateJsErrors.length} error(s)`,
           };
+          console.log("validateJsErrors...", validateJsErrors);
 
           let errors = {};
 
           validateJsErrors.map((d) => {
-            errors[d.attribute] = d.error;
+            if (!errors[d.attribute]) {
```
Comment (ShubhamJoshi-1994) [inline app/requests/base.js:42]: @Subbu-betalectic can you please write here and explain to me this piece of code you have implemented?

File: app/requests/createGmProductsConstraints.js [hunk 2]
```diff
@@ -88,58 +88,51 @@ const constraints = {
   name: {
     presence: {
       allowEmpty: false,
-      message: "^Please enter a Product Name",
+      message: "^Enter a Product Name",
```
Comment (ShubhamJoshi-1994) [inline app/requests/createGmProductsConstraints.js:91]: `Enter Product Name` and not `Enter a Product Name`.

@Subbu-betalectic please match these names with what you have written in the mars issue.
https://mars.betalectic.com/apps/product-wiredup/wiredup/-/issues/388

**Review by ShubhamJoshi-1994 — CHANGES_REQUESTED** [review]
@Subbu-betalectic please work on the feedback

File: app/requests/base.js (hunk 1 above)
Comment (Subbu-betalectic) [reply app/requests/base.js:42]: @ShubhamJoshi-1994 In payload if the currency goes empty, We have two validation rules so we are getting two error messages. Earlier we are getting a second error message.

Screenshot 2022-10-17 at 3 36 07 PM
Screenshot 2022-10-17 at 3 30 53 PM



now I have added a check to whether the error key is already present in the errors object then it won't overwrite existing error message
Screenshot 2022-10-17 at 3 42 16 PM
Screenshot 2022-10-17 at 3 42 39 PM

File: app/requests/createGmProductsConstraints.js (hunk 2 above)
Comment (Subbu-betalectic) [reply app/requests/createGmProductsConstraints.js:91]: Ok @ShubhamJoshi-1994

File: app/requests/base.js (hunk 1 above)
Comment (ShubhamJoshi-1994) [reply app/requests/base.js:42]: ok @Subbu-betalectic , it's clear to me now. This should work.

**Review by ShubhamJoshi-1994 — CHANGES_REQUESTED** [review]
LGTM, merging.

**Pull request merged**